Update webpack peerDependency with rc

**I'm submitting a bug report**

**Webpack version:**
2.2.0-rc.0

**Webpack Karma version**:
1.8

**Current behavior:**

PeerDependency failure

**Expected/desired behavior:**

exit 0

npm3 and yarn only warn
```
npm WARN karma-webpack@1.8.0 requires a peer of webpack@^1.1.0 || ^2 || ^2.1.0-beta but none was installed.
```
```
[3/4] Linking dependencies...
warning Incorrect peer dependency "webpack@^1.1.0 || ^2 || ^2.1.0-beta".
```

Here's the npm2
```
npm WARN peerDependencies The peer dependency webpack@^1.1.0 || ^2 || ^2.1.0-beta included from 
npm ERR! Windows_NT 6.3.9600
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package webpack@2.2.0-rc.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer karma-webpack@1.8.0 wants webpack@^1.1.0 || ^2 || ^2.1.0-beta

npm ERR! Please include the following file with any support request:
npm ERR!     ***\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "url": "https://github.com/webpack/karma-webpack.git"
   },
   "peerDependencies": {
-    "webpack": "^1.1.0 || ^2 || ^2.1.0-beta"
+    "webpack": "^1.1.0 || ^2 || ^2.1.0-beta || ^2.2.0-rc"
   },
   "dependencies": {
     "async": "~0.9.0",

